### PR TITLE
fix(config): update Philio PST02A and PST02B to use partial config parameters (5, 6, 7)

### PR DESCRIPTION
**Pull request by orandasoft**
- Remove supportsZWavePlus: true
- Update configuration parameters 5, 6, and 7 to use partial parameters
- Add meta section with description for include, exclude, and reset

### CODE REVIEW

**Comment by orandasoft** [comment]
error file=packages/config/config/devices/0x013c/pst02b.json::Parameter #6 has a single bit mask defined. Either add more, or delete the bit mask.

PST02A, PST02B, and PST02C all use the same configuration definition. PST02B just happens to support only one bit. Shouldn't we use the same code for consistency?

**Comment by orandasoft** [comment]
@zwave-js-bot fix lint

**Comment by AlCalzone** [comment]
> Shouldn't we use the same code for consistency?

Ideally yes, but this would require a much larger change and the linter doesn't know this bitmask is shared with other devices.

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Looks good, but all the new partials with just 2 options should have
```
"allowManualEntry": false
```

and that single bitmask for param 6 in the one device needs to be a non-partial.

**Comment by orandasoft** [comment]
> ```
> "allowManualEntry": false
> ```

Where should I put this?

**Comment by AlCalzone** [comment]
> Where should I put this?

Before each of these lines:
```
"options": [
```
(where the options define ALL the possibilities)

File: packages/config/config/devices/0x013c/pst02a.json [hunk 1]
```diff
@@ -61,47 +60,309 @@
...
 		},
-		"7": {
-			"label": "Customer Function",
-			"description": "Enable or disable functions of the multisensor.",
+		"5[0x08]": {
+			"label": "Temperature Scale",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 127,
-			"defaultValue": 20,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x013c/pst02a.json:96]: The old parameter 7 had a default value of 20, which means that the default for 7[0x10] and 7[0x04] should be 1 if I'm not mistaken.

File: packages/config/config/devices/0x013c/pst02a.json [hunk 2]
```diff
@@ -61,47 +60,309 @@
...
 		},
-		"6": {
-			"label": "Multi-Sensor Function Switch",
-			"description": "Enable or disable functions of the multisensor.",
+		"5[0x04]": {
+			"label": "Door/Window Function",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 127,
-			"defaultValue": 4
```
Comment (AlCalzone) [inline packages/config/config/devices/0x013c/pst02a.json:88]: The old parameter 6 had a default value of 4, so 6[0x04] should have a default value of 1, the rest 0 (if I'm not mistaken).

File: packages/config/config/devices/0x013c/pst02b.json [hunk 3]
```diff
@@ -38,37 +37,183 @@
...
 			"minValue": 0,
-			"maxValue": 127,
-			"defaultValue": 4
+			"maxValue": 2,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 2
```
Comment (AlCalzone) [inline packages/config/config/devices/0x013c/pst02b.json:122]: This doesn't match the old default value of 4. Which one is correct?

File: packages/config/config/devices/0x013c/pst02b.json [hunk 4]
```diff
@@ -38,37 +37,183 @@
...
+					"value": 1
+				}
+			]
+		},
+		"7[0x04]": {
+			"label": "PIR Super Sensitivity Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x013c/pst02b.json:153]: This doesn't match the old default value of 0.

**Review by AlCalzone — COMMENTED** [review]
I'm not sure about the new default values. They don't always match the old ones, but the old ones could also be wrong.

File: packages/config/config/devices/0x013c/pst02a.json (hunk 1 above)
Comment (orandasoft) [reply packages/config/config/devices/0x013c/pst02a.json:96]: According to this document, the default value is 4: https://doc.eedomus.com/files/PST02_Manual-2014-09-17.pdf

File: packages/config/config/devices/0x013c/pst02a.json (hunk 2 above)
Comment (orandasoft) [reply packages/config/config/devices/0x013c/pst02a.json:88]: Changed default value to 0 for 6[0x20]

File: packages/config/config/devices/0x013c/pst02b.json (hunk 3 above)
Comment (orandasoft) [reply packages/config/config/devices/0x013c/pst02b.json:122]: PST02B only supports one bit with default value 0: https://doc.eedomus.com/files/PST02_Manual-2014-09-17.pdf

File: packages/config/config/devices/0x013c/pst02b.json (hunk 4 above)
Comment (orandasoft) [reply packages/config/config/devices/0x013c/pst02b.json:153]: According to this document, the default value for parameter 7 is 4: https://doc.eedomus.com/files/PST02_Manual-2014-09-17.pdf

**Review by AlCalzone — APPROVED** [review]

**Comment by AlCalzone** [comment]
Thanks!

**Pull request merged**